### PR TITLE
Be# 7. get all users from db#21

### PR DESCRIPTION
**Pull request by Aspink**

### CODE REVIEW

File: backend/src/main/java/MapBravoApp.java
Comment (MateuszLesisz) [inline backend/src/main/java/MapBravoApp.java:1]: all logic should be done in services

Comment (MateuszLesisz) [inline backend/src/main/java/MapBravoApp.java:1]: all logic should be done in services

File: backend/src/main/java/service/User.java [hunk 1]
```diff
@@ -1,3 +1,5 @@
+package service;
```
Comment (MateuszLesisz) [inline backend/src/main/java/service/User.java]: Model classes should not be in service packs. It's better to create a separate "model" package or keep all classes responsible for the user in the "user" package

File: backend/src/main/java/service/UserService.java [hunk 2]
```diff
@@ -2,22 +2,48 @@
 
 import configuration.DatabaseConnection;
 
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
+//import java.sql.Statement;
```
Comment (MateuszLesisz) [inline backend/src/main/java/service/UserService.java:7]: unused import ?

File: backend/src/main/java/service/UserService.java [hunk 3]
```diff
@@ -2,22 +2,48 @@
...
+            while (resultSet.next()) {
+                String nickname = resultSet.getString(2);
+                String city = resultSet.getString(3);
+                String zipCode = resultSet.getString(4);
+                String country = resultSet.getString(5);
+
+                User user = new User(nickname, city, zipCode, country);
+                users.add(user);
+            }
+
```
Comment (MateuszLesisz) [inline backend/src/main/java/service/UserService.java:41]: empty line

File: backend/src/main/java/service/UserService.java (hunk 2 above)
Comment (Aspink) [reply backend/src/main/java/service/UserService.java:7]: Replaced by import java.sql.* (this 2 and more needed).

File: backend/src/main/java/service/UserService.java (hunk 3 above)
Comment (Aspink) [reply backend/src/main/java/service/UserService.java:41]: Removed

File: backend/src/main/java/MapBravoApp.java
Comment (Aspink) [reply backend/src/main/java/MapBravoApp.java:1]: I think this "logic" in MapBravoApp.java is temporary just for testing. Now inactivated as comment.

File: backend/src/main/java/service/User.java (hunk 1 above)
Comment (Aspink) [reply backend/src/main/java/service/User.java]: Created "model" package. User moved to it.

File: backend/src/main/java/service/UserService.java [hunk 4]
```diff
@@ -16,7 +17,7 @@ public void addUser(String addUserQuery) {
             Statement statement = databaseConnection.connection();
             statement.execute(addUserQuery);
             statement.close();
-            System.out.println("service.User has been correctly added.");
+            System.out.println("Model.User has been correctly added.");
```
Comment (MateuszLesisz) [inline backend/src/main/java/service/UserService.java]: "User has been correctly added" -> there is no need to write the package name

File: backend/src/main/java/service/UserService.java [hunk 5]
```diff
@@ -29,18 +30,15 @@ public ArrayList<User> getUsers() {
...
                 String zipCode = resultSet.getString(4);
                 String country = resultSet.getString(5);
-
                 User user = new User(nickname, city, zipCode, country);
                 users.add(user);
             }
 
-
         } catch (SQLException e) {
             System.err.println("Błąd połączenia z bazą danych: " + e.getMessage());
```
Comment (MateuszLesisz) [inline backend/src/main/java/service/UserService.java]: Translate it plz

File: backend/src/main/java/service/UserService.java (hunk 4 above)
Comment (Aspink) [reply backend/src/main/java/service/UserService.java]: Automatic refactoring made it! Removed.

File: backend/src/main/java/service/UserService.java (hunk 5 above)
Comment (Aspink) [reply backend/src/main/java/service/UserService.java]: Translated.

**Review by MateuszLesisz — APPROVED** [review]

**Pull request merged**